authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationHDFSPlugin.java

```diff
@@ -183,36 +540,66 @@ public List<AuthorizationSecurableObject> translatePrivilege(SecurableObject sec
                   // in the RangerAuthorizationHDFSPlugin.
                   break;
                 case USE_SCHEMA:
+                  switch (securableObject.type()) {
+                    case METALAKE:
+                    case CATALOG:
+                    case SCHEMA:
+                      AuthorizationUtils.getMetadataObjectLocation(
+                              identifier, MetadataObjectUtil.toEntityType(securableObject))
+                          .stream()
+                          .forEach(
+                              locationPath -> {
+                                PathBasedMetadataObject pathBaseMetadataObject =
+                                    new PathBasedMetadataObject(
+                                        securableObject.parent(),
+                                        securableObject.name(),
+                                        locationPath,
+                                        PathBasedMetadataObject.Type.PATH);
+                                pathBaseMetadataObject.validateAuthorizationMetadataObject();
+                                rangerSecurableObjects.add(
+                                    generateAuthorizationSecurableObject(
+                                        pathBaseMetadataObject.names(),
+                                        locationPath,
+                                        PathBasedMetadataObject.Type.PATH,
+                                        rangerPrivileges));
+                              });
+                      break;
+                    default:
+                      checkOmissionTranslate(
```

Comment:
`Omission`? This is not a clear name.

Reply:
OK, change to `checkUnprocessedTranslate()`

Reply:
I don't get the meaning of this check.

Reply:
I removed it.